pkg/secrethub/credential.go

```diff
@@ -321,31 +325,77 @@ func (c RSACredential) Decoder() CredentialDecoder {
 }
 
 // Wrap encrypts data, typically an account key.
-func (c RSACredential) Wrap(plaintext []byte) (crypto.CiphertextRSAAES, error) {
-	return c.RSAPrivateKey.Public().Encrypt(plaintext)
+func (c RSACredential) Wrap(plaintext []byte) (*api.EncryptedData, error) {
+	ciphertext, err := c.RSAPrivateKey.Public().Encrypt(plaintext)
+	if err != nil {
+		return nil, err
+	}
+	key := api.NewEncryptionKeyEncrypted(
+		crypto.SymmetricKeyLength*8,
```

Comment:
This looks pretty specific to do here. Can we create a function `ciphertext.ToEncryptedData()` instead, e.g. in the `crypto` package?

Reply:
Doesn't that lead to a circular dependency?

Reply:
Hmm ok I agree with your rule of thumb to keep the two separate. Don't like the hard coded 8 here though.